docs/docs/cmd/teams/conversationmember/conversationmember-add.md

```diff
@@ -0,0 +1,54 @@
+# teams conversationmember add
+
+Adds or sets a conversationmember in a private channel
+
+## Usage
+
+```sh
+m365 teams conversationmember add [options]
+```
+
+## Options
+
+`-i, --teamId [teamId]`
```

Comment:
The short `-i` was not a part of the spec. I believe we decided not to do it, to disambiguate between `teamId` and `channelId` which could theoretically both be referred to by `-i`. Let's remove it to stay in sync.

Reply:
Sure @waldekmastykarz, I can remove the short `-i` and `-c` from this command. However, since they are present in the [conversationmember list](https://pnp.github.io/cli-microsoft365/cmd/teams/conversationmember/conversationmember-list/) command as well, should we raise a new issue to remove them from there to keep both these commands aligned?

Reply:
Ah missed that, sorry. In that case, let's update the spec to include them. Makes sense to keep both commands aligned.